src/codemodder/codemods/regex_transformer.py

```diff
@@ -0,0 +1,60 @@
+import re
+from typing import Pattern
+
+from codemodder.codemods.base_transformer import BaseTransformerPipeline
+from codemodder.codetf import Change, ChangeSet
+from codemodder.context import CodemodExecutionContext
+from codemodder.diff import create_diff
+from codemodder.file_context import FileContext
+from codemodder.logging import logger
+from codemodder.result import Result
+
+
+class RegexTransformerPipeline(BaseTransformerPipeline):
+
+    def __init__(self, pattern: Pattern, replacement: str, change_description: str):
```

Comment:
We don't need to overengineer this right now but I can imagine scenarios where we'd like a more sophisticated callback-based replacement. `re.sub` already supports `repl` as a string or a function, so maybe we want to capture that in the type signature?